src/lib.rs

```diff
@@ -416,6 +417,20 @@ mod document_encrypt_result {
     }
 }
 
+mod document_edek_encrypt_result {
+    use super::*;
+
+    pub fn id(d: &DocumentDetachedEncryptResult) -> DocumentId {
+        d.id().clone()
+    }
+    pub fn encrypted_data(d: &DocumentDetachedEncryptResult) -> Vec<i8> {
```

Comment:
I realize that we already botched this with the `document_encrypt_result` types, but if we wanted to look like actual crypto people, we would call this `ciphertext`. Anybody have an opinion on whether we should change that or just feel bad and leave it as is? @coltfred @clintfred

Reply:
I don't have any strong feelings about this. To me, both communicate clearly what you are getting back. If we change this here, I would recommend making similar changes in `ironoxide` as well.

Reply:
I'd be fine with changing it, as it is more standard. Doing it because it's what "Crypto people" would do leaves a bad taste in my mouth as most crypto libraries have unclear apis that are painful to use and easy to screw up, but the word "Ciphertext" is probably standard enough to switch. Clint is right that we should just change it in all the places if we're going to do it, though.

Reply:
To be clear, I'm only advocating for changing it if we change it everywhere probably, meaning breaking the SDK all the way down to IronOxide.

Reply:
`ciphertext` looks more like something the cool kids would do.

Reply:
Ticket created to decide/work on this: https://github.com/IronCoreLabs/ironoxide/issues/31